utils/not_doctested.txt

```diff
@@ -209,6 +209,7 @@ docs/source/en/model_doc/roc_bert.md
 docs/source/en/model_doc/roformer.md
 docs/source/en/model_doc/rwkv.md
 docs/source/en/model_doc/sam.md
+docs/source/en/model_doc/seamless_m4t.md
```

Comment:
is this too large and can't even be tested with 16GB GPU?
if it is just slow and fail on CircleCI, we can put it in `slow_documentation_tests.txt` instead

Reply:
Just slow I believe, I'll move it to the right file than